client-spark/spark3/src/main/java/org/apache/spark/shuffle/DelegationRssShuffleManager.java

```diff
@@ -248,6 +248,7 @@ public boolean unregisterShuffle(int shuffleId) {
   @Override
   public void stop() {
     delegate.stop();
+    coordinatorClients.forEach(CoordinatorClient::close);
```

Comment:
spark2 should have the same problem?

Reply:
> spark2 should have the same problem?

You are right, I will fix it in spark2 as well.